WcaOnRails/app/models/user.rb

```diff
@@ -496,35 +496,36 @@ def editable_fields_of_user(user)
       return fields
     end
     if user == self
-      fields << :current_password
-      fields << :password << :password_confirmation
-      fields << :email
-      fields << :preferred_events
+      fields += %i(
+        current_password password password_confirmation
+        email preferred_events results_notifications_enabled
+      )
       fields << { user_preferred_events_attributes: [:id, :event_id, :_destroy] }
-      fields << :results_notifications_enabled
     end
     if admin? || board_member?
-      fields << :delegate_status
-      fields << :senior_delegate_id
-      fields << :region
+      fields += %i(delegate_status senior_delegate_id region)
+    end
+    if user.any_kind_of_delegate? && (user == self || user.senior_delegate == self || admin? || board_member)
+      fields += %i(location_description phone_number notes)
     end
     if admin? || any_kind_of_delegate?
-      fields << :wca_id << :unconfirmed_wca_id
-      fields << :avatar << :avatar_cache
+      fields += %i(
+        wca_id unconfirmed_wca_id
+        avatar avatar_cache
+      )
     end
     if user == self || admin? || any_kind_of_delegate?
       cannot_edit_data = !!cannot_edit_data_reason_html(user)
       if !cannot_edit_data
-        fields << :name
-        fields << :dob
-        fields << :gender
-        fields << :country_iso2
+        fields += %i(name dob gender country_iso2)
```

Comment:
Not sure what direction is "up", but concretely, you start a story by pulling the code from Github. When done 4-16 hours later, you `push` it back (up?). There are occasional merge conflicts, but since neither of us changed a ton of code, and it's fresh in both our minds, it's almost always trivially resolved.

The part that baffles most people is that we only make branches as rare exceptions. 98% of the work is done on `master`. :)

I don't really understand why so many organizations keep making merge hells by working on PRs for weeks or months.